css/styles.css

```diff
@@ -50,3 +50,37 @@ textarea {
   border-radius: 5px;
   margin-top: 1.3rem;
 }
+
+ul {
+  list-style: none;
+  padding-inline-start: 0;
+  position: relative;
+}
+
+.flex {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.new-btn {
+  display: block;
+  display: flex;
```

Comment:
You shouldn't have 2 different `display` properties. The one that comes last is just going to override the other one.